recipe/meta.yaml

```diff
@@ -26,38 +24,41 @@ requirements:
     - wget
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
+    - libtool
   host:
     - python
     - pip
     - gsl
     - fftw
     - cfitsio
-    - libsharp
     - chealpix
     - numpy
   run:
     - python
     - gsl
     - fftw
-    - libsharp
     - chealpix
     - numpy
 
 test:
-  source_files:
-    - test/*
-  requires:
-    - healpy
   imports:
     - pymaster
+  requires:
+    - healpy
+    - pytest
+  source_files:
+    - pymaster/tests/*
+    - test/*
   commands:
-    - namaster -h
+    - pytest -vv pymaster/tests
 
 about:
   home: https://github.com/LSSTDESC/NaMaster
   license: BSD-3-Clause
   license_family: BSD
-  license_file: LICENSE
+  license_file:
+    - LICENSE
+    - COPYING
```

Comment:
You need to commit this file from libsharp2 to the feedstock at namaster-feedstock/recipe/COPYING.

You also need to adjust the license indicator to reflect the license of libsharp2 as well.

Reply:
Oops, forgot to include that. Done!

Reply:
> You also need to adjust the license indicator to reflect the license of libsharp2 as well.
Actually, I just realized I'm not sure how to do this, and I couldn't find anything online.

But this leads me to a more basic question (which maybe I should have started with). libsharp is GPL-2, whereas namaster is BSD-3. I think they are compatible (I believe the problem is with GPL-3), so is this a problem?

Reply:
Seems fine.

Reply:
Thanks!